_data/kamamile-tea/kamamile-tea.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Kamille Tipan # No longer than 28 characters
+institution: Francis Lewis High School 🚩 # no longer than 58 characters
+profile_pic: kamamile-tea.jpg # Name and extension of your profile picture(ex. mona.png) The picture must be squared and 544px on width and height.
+quote: Shout out to all the gay cousins who gotta carry the weight of being the best family member # no longer than 100 characters, avoid using quotes(") to guarantee the format remains the same.
+github_user: Kamamile-tea
```

Comment:
Hi @kamamile-tea, It appears that the first letter here is capital - which is different from your username. A user previously had an issue with this, I would suggest that you make it perfectly like your GitHub username (with the same capitalization) to avoid any issues.

Reply:
Okay thank you! I will change that now!